lib/core/src/receive_swap.rs

```diff
@@ -415,7 +473,8 @@ impl ReceiveSwapHandler {
     async fn verify_lockup_tx(
         &self,
         receive_swap: &ReceiveSwap,
-        swap_update_tx: &SwapUpdateTxDetails,
+        tx_id: &str,
+        tx_hex: &str,
         verify_confirmation: bool,
     ) -> Result<()> {
```

Comment:
Shouldn't we also verify the incoming amount here?

Reply:
I'm not sure what the recourse here should be. We can't help Boltz refund, i guess leave the swap to expire